File: Searching/linear_search/linear_search.py



def linearSearch(arr, val):
    for i in range(len(arr)):
        if arr[i] == val:
            return i
    return -1



print(linearSearch([1, 5, 7, 9, 3], 7))
print(linearSearch([1, 5, 7, 9, 3], 2))

File: Searching/linear_search/linear_search.js



function linearSearch(arr, val){
    for(let i = 0; i < arr.length; i++){
        if(arr[i] === val) return i;
    }
    return -1
}



console.log(linearSearch([1, 3, 7, 9, 5], 7))
console.log(linearSearch([1, 3, 7, 9, 5], 2))
